run_pylint.py

```diff
@@ -0,0 +1,162 @@
+#                                                         -*- coding: utf-8 -*-
+# Probe directory tree for python files and pass them to pylint
+# Copyright (C) 2019  Red Hat, Inc.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
```

Comment:
Please add an SPDX license identifier, this one seems to be correct here:
```# SPDX-License-Identifier: GPL-3.0-or-later```

Reply:
Unless it is based on another GPL-licensed source, I would prefer to use the MIT license (sorry for not having spotted it earlier)

Reply:
I prefer MIT as well, GPL 3.0 was used only because it is the same license that is  used in [linux-system-roles/template](https://github.com/linux-system-roles/template). The license was changed to MIT now.